sdk/include/psdk/winerror.h

```diff
@@ -18,6 +18,9 @@
 
 #ifndef __WINE_WINERROR_H
 #define __WINE_WINERROR_H
+#ifndef __REACTOS__
```

Comment:
Have you checked whether compilation fails if you keep the specstrings inclusion even for ReactOS? If not, you can keep it.